### PR TITLE
Add a more understandable msg on missing command (ufw)

### PR DESCRIPTION
**Pull request by jbarotin**
##### SUMMARY

Replace message "Not any of the command arguments {} given" by the standard Ansible required_one_of.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

ufw

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/system/ufw.py [hunk 1]
```diff
@@ -348,7 +348,8 @@ def ufw_version():
     commands = dict((key, params[key]) for key in command_keys if params[key])
 
     if len(commands) < 1:
-        module.fail_json(msg="Not any of the command arguments %s given" % commands)
+        msg = "You must provide one these commands '{}'".format("','".join(command_keys))
```
Comment (mkrizek) [inline lib/ansible/modules/system/ufw.py]: Maybe pass `required_one_of` parameter to `AnsibleModule` instead?

Comment (jbarotin) [reply lib/ansible/modules/system/ufw.py]: Thanks for the suggestion :) I update my PR in this way !

**Comment by ovcharenko** [comment]
LGTM
shipit

**Comment by mkrizek** [comment]
@jbarotin please resolve the conflicts, thanks!

File: lib/ansible/modules/system/ufw.py [hunk 2]
```diff
@@ -344,11 +348,8 @@ def ufw_version():
     params = module.params
 
     # Ensure at least one of the command arguments are given
```
Comment (mkrizek) [inline lib/ansible/modules/system/ufw.py]: You can delete this comment I think.

**Review by gundalow — APPROVED** [review]
PR looks good
https://docs.ansible.com/ansible/devel/dev_guide/developing_rebasing.html may help you rebase

**Comment by jbarotin** [comment]
After it seems it's fixed :)

**Pull request closed without merging**